### ISSUE TITLE
How should people without a GitHub account contact us?

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#16]
When someone without a GitHub account wants to content us, for example to report an error or to ask a question, how would that work?

Some possibilities
- ask in Slack #accessibility-docs channel on Make WordPress
- reach out in Mastodon or X
- post a comment somewhere in make.wp.org/accessibility 
- we create and publish an email address for the project (and take the spam for granted)
- we create an option for a contact form (in make.wp.org/accessibility?)
- any other ideas?

See also on make/accessibility: [How to get in touch](https://make.wordpress.org/accessibility/handbook/get-involved/#how-do-you-get-in-touch).

I think we can expect accessibility experts from outside the WordPress bubble to want to reach out to us.

**Comment by rianrietveld** [comment]
Can you make it so that you don't have to login into WP.org first?

Fields:
- name
- email
- question or remark
